jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1118,26 +1127,20 @@ public void testGetTaskAssignedAsBusinessAdminPaging() {
     
     @Test
     public void testGetTaskAssignedAsBusinessAdminByStatus() {
-
-        for (int i = 0; i < 10; i++) {
-
-            processService.startProcess(deploymentUnit.getIdentifier(), "org.jbpm.writedocument");
-        }
+        startProcesses(10, "org.jbpm.writedocument");
         
         List<Status> statuses = new ArrayList<Status>();
         statuses.add(Status.Ready);
         statuses.add(Status.Reserved);
 
         List<TaskSummary> tasks = runtimeDataService.getTasksAssignedAsBusinessAdministratorByStatus("Administrator", statuses, new QueryFilter(0, 5));
-        assertNotNull(tasks);
-        assertEquals(5, tasks.size());
```

Comment:
Why was this removed? It should still hold I think.

Reply:
There was an already existing issue with this test. 
Basically Task table is not removed between test, which leads to inconsistencies. 
Since it is apparently not related with these changes, Gonzalo will investigate this as a separate task.